### PR TITLE
Add example for `undefined-all-variable` message documentation

### PR DESCRIPTION
**Pull request by gunungpw**
- [x] Write a good description on what the PR does.
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description
- add example for `undefined-all-variable`
- add link for Python docs about `import *` Package (usage of `__all__`)

Related: #5953

### CODE REVIEW

File: doc/data/messages/u/undefined-all-variable/bad.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "age",
+    "get_fruit_color",  # [undefined-all-variable]
+]
+
+pet = "Cat"
+age = 10
+
+
+def get_pet_age():
+    pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/u/undefined-all-variable/bad.py:12]:
```suggestion
__all__ = ["get_fruit_colour"]  # [undefined-all-variable]

def get_fruit_color():
    pass
```

File: doc/data/messages/u/undefined-all-variable/good.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "age",
+    "get_pet_age",
+]
+
+pet = "Cat"
+age = 10
+
+
+def get_pet_age():
+    pass
```
Comment (Pierre-Sassoulas) [inline doc/data/messages/u/undefined-all-variable/good.py:12]:
```suggestion
__all__ = ["get_fruit_color"]

def get_fruit_color():
    pass
```

**Review by Pierre-Sassoulas — COMMENTED** [review]
I think we can keep only one import in ``__all__``.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Comment by gunungpw** [comment]
Yep, agree 👍

**Pull request merged**